controllers/clustersummary_watchers.go

```diff
@@ -22,6 +22,9 @@ import (
 	"sync"
 
 	"github.com/go-logr/logr"
+	"github.com/projectsveltos/libsveltos/lib/logsettings"
```

Comment:
thank you for pushing PR. Can you please run "make fmt" which internally with run a go imports ? Thank you

Reply:
pushed the change after `make fmt`